package ExceptionHandling;

public class ExceptionHandle {

	public static void main(String[] args) {
		System.out.println("Here i am writing exception handling program:");
		int a =10;
		int b =0;
		
		try {
			int result = a/b;
			System.out.println("Division result is: "+result);
		}
		catch(ArithmeticException ae){
			System.out.println(ae.toString());
		}
		finally {
			System.out.println("This finally block will execute all time");
		}

	}

}
